tests/test_features/test_histogram.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from tests.base import VisualTestCase
+from tests.dataset import DatasetMixin
+from yellowbrick.features.histogram import *
+
+##########################################################################
+## BalancedBinningReference Tests
+##########################################################################
+
+class TestBalancedBinningReference(VisualTestCase, DatasetMixin):
+	"""
+	Test the BalancedBinningReference visualizer 
+	"""
+
+	def test_balancedbinningreference(self):
+		"""
+		Test Histogram on a real dataset
+		"""
+		# Load the data from the fixture
+		dataset = self.load_data('occupancy')
+
+		# Get the data
+		X = dataset["temperature"]
+
+		
+		visualizer = BalancedBinningReference()
+		visualizer.fit(X)
+		visualizer.poof()
+		self.assert_images_similar(visualizer)
```

Comment:
This is a very good test; thank you for including it! For the `assert_images_similar` to pass, an image needs to be added to `tests/baseline_images/test_target/test_binnning/test_balancedbinningreference.png` -- you can add this image by first running the test, then using the sync script from the root of the project:

```
$ python -m tests.images test/test_target/test_binning.py 
```

This will move all images from the the `tests/actual_images/` folder (generated during testing) to `tests/baseline_images/`. Please make sure the baseline images are committed to the repository (and not the actual images).